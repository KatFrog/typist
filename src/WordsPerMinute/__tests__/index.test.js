import React from 'react';
import WordsPerMinute from '../';
import renderer from 'react-test-renderer';

describe('WordsPerMinute', () => {
  it('renders correctly', () => {
    const wpm = [10, 20];
    const component = renderer.create(
      <WordsPerMinute wordsPerMinute={wpm} />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
