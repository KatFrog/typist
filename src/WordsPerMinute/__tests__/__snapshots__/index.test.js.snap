// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WordsPerMinute renders correctly 1`] = `
<pre>
  [
  10, 20
  ]
</pre>
`;
